pkg/operator/secretannotator/aws/reconciler.go

```diff
@@ -67,6 +87,23 @@ func Add(mgr manager.Manager, r reconcile.Reconciler) error {
 	if err != nil {
 		return err
 	}
+
+	// Watch for changes to CCO config
+	configPredicate := predicate.Funcs{
```

Comment:
This watch should be added to the secretannotator for the other platforms as well. Extract this into a function that can be called by all of the secretannotators.

Reply:
done